example/src/screens/expandableCalendar.tsx

```diff
@@ -190,6 +190,7 @@ export default class ExpandableCalendarScreen extends Component<Props> {
         <AgendaList
           sections={ITEMS}
           renderItem={this.renderItem}
+          // scrollToFutureEvents
```

Comment:
Naming is a bit confusing. How about something like 'scrollToNextMarkedDate' or 'scrollToNextEvent'?
Also, You need to adjust the example to show the feature. I don't see it working in the example...

Reply:
I updated the prop name.
The example works for me, try to uncomment this line and press on the next date in the weekCalendar

Reply:
I did... But when I press the 3rd of the month it doesn't scroll to the 4th, where there is content...

Reply:
Sorry. It works now. Probably cache...